parsec/backend/invite.py

```diff
@@ -305,6 +320,7 @@ def _to_http_redirection_url(client_ctx, invitation):
                     reply_to=reply_to,
                     organization_id=client_ctx.organization_id,
                     invitation_url=_to_http_redirection_url(client_ctx, invitation),
+                    parsec_url=_to_url(client_ctx, invitation),
```

Comment:
why not provide `invitation_addr` instead of `invitation_url`&`parsec_url` ? (so we could have a single `_to_invitation_addr` function)